### ISSUE TITLE
Not respecting $TERM or $NO_COLOR

### ISSUE THREAD

**Issue by robnagler** [rclone/rclone#6604]
#### The associated forum post URL from `https://forum.rclone.org`

https://forum.rclone.org/t/specifying-preservation-of-ansi-codes-in-rclone-output/17305

#### What is the problem you are having with rclone?

`rclone config --color=never` is not allowed

#### What is your rclone version (output from `rclone version`)

1.60.1

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)

Centos 7, 64 bit

#### Which cloud storage system are you using? (e.g. Google Drive)

Google Drive

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)

`rclone config --color=never`

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)

```txt
~$ rclone config
2022/12/05 23:37:31 NOTICE: Config file "/home/nagler/.config/rclone/rclone.conf" not found - using defaults
No remotes found, make a new one?
n) New remote
s) Set configuration password
q) Quit config
n/s/q> n

Enter name for new remote.
name> radiasoft

Option Storage.
Type of storage to configure.
Choose a number from below, or type in your own value.
^[[91m 1 / 1Fichier
   \ (fichier)
^[[0m^[[92m 2 / Akamai NetStorage
   \ (netstorage)
^[[0m^[[91m 3 / Alias for an existing remote
   \ (alias)
^[[0m^[[92m 4 / Amazon Drive
   \ (amazon cloud drive)
```

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.61

Thank you @kverstae

**Comment by robnagler** [comment]
Thank you!
